sound/soc/sof/ipc4-pcm.c

```diff
@@ -21,12 +21,18 @@
  * @dai_copier: the dai copier of the pcm stream
  * @stream_start_offset: reported by fw in memory window
  * @llp_offset: llp offset in memory window
+ * @boundary: wrap boundary should be used for the LLP frame counter
+ * @delay: Calculated and stored in pointer callback to be returned as delay
+ *	   value
```

Comment:
suggested edit
"Calculated and stored in pointer callback. The stored value is returned in the delay callback."

Reply:
OK